pypolestar/cli.py

```
@@ -0,0 +1,71 @@
+import argparse
+import asyncio
+import json
+import logging
+from getpass import getpass
+
+from . import PolestarApi
+from .exceptions import PolestarApiException, PolestarAuthException
```

Ruff (0.8.2)

`.exceptions.PolestarApiException` imported but unused

Remove unused import: `.exceptions.PolestarApiException`

(F401)